### ISSUE TITLE
encrypted rooms

### ISSUE THREAD

**Issue by seshubonam** [Stvad/matrix-serverless-setup#1]
would it work in encrypted rooms?

**Comment by Stvad** [comment]
Yes, though you'd receive an encrypted version of the message to the handler function and would need to decrypt it yourself. 

Some meta messages are plain text even in encrypted room (e.g. reacts to encrypted messages are not themselves encrypted 🤷‍♂️) so there is some useful things you can automate without setting up decryption (e.g. snoozing message based on a react to it)

**Comment by seshubonam** [comment]
super thanks. will try this out and update.
